cairo/src/matrices.rs

```diff
@@ -2,20 +2,11 @@
 
 use crate::error::Error;
 use crate::utils::status_to_result;
-use libc::c_double;
 
-#[repr(C)]
-#[derive(Debug, Clone, Copy, PartialEq)]
-pub struct Matrix {
-    pub xx: c_double,
-    pub yx: c_double,
-
-    pub xy: c_double,
-    pub yy: c_double,
-
-    pub x0: c_double,
-    pub y0: c_double,
-}
+#[repr(transparent)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
@bilelmoussaoui What do you think about implementing `Copy` or not here? For all the graphene types (which are similar) we implement `Copy`, and e.g. a `[f64; 16]` would also be `Copy` (and that would be a `cairo::Matrix` with a different name).

Reply:
I guess that is fine, but then will all the Matrix methods take the type by ref or not? because the graphene types do

Reply:
You mean leave it like this or add the `Copy` impl back? And taking any value that is bigger than a pointer by reference makes sense.

Reply:
Add the copy impl back

Reply:
Considering that we need to make our methods take `self` behind a reference, I think it's pretty obvious that `Matrix` shouldn't implement `Copy`.

Reply:
Those are completely independent concerns

Reply:
In what way? If we need to be careful about it, our users will need to. At this point, it's simply better to prevent this problem by making it impossible to happen.

Reply:
Copying comment from @ranfdev from Matrix:
> When the CPU fetches some data from the RAM, multiple words are fetched and put in the cache. On a modern computer fetching 64 bits should take mostly the same time as fetching 256 bits (given that those bits are contiguous in memory).

The only real concern here is for passing via functions as passing anything bigger than a pointer can easily require some additional work: making space available on the stack and copying the value in there so that a pointer to the stack can be passed to the function.